packages/smart/contracts/libraries/ICash.sol

```diff
@@ -1,8 +1,9 @@
-pragma solidity 0.5.15;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
 
 import "./IERC20.sol";
 
 
-contract ICash is IERC20 {
-    function faucet(uint256 _amount) public returns (bool);
+abstract contract ICash is IERC20 {
```

Comment:
should be an interface

Reply:
@robert-e-davidson3 Should ICash exist at all?

Reply:
I retract that question.